FreeRTOS/FreeRTOS-Plus-TCP
# IPv6/Multi: Reduced complexity FreeRTOS_RA.c

Description
-----------
FreeRTOS_RA had two functions that were too complex. I made them simpler by separating some of the code in helper functions:

vReceiveRA()
- vReceiveRA_ReadReply()

vRAProcess()
- xRAProcess_HandleWaitStates()
- xRAProcess_HandleOtherStates()

I also merged the changes proposed in #271, which can now be closed.

Test Steps
-----------
I tested the sources by running the RA procedure.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
